packages/business-rules/src/validation/appeal/decision-date/is-within-decision-date-expiry-period.js

```diff
@@ -10,13 +11,13 @@ const isValid = require('../../generic/date/is-valid');
  * @param {Date} now
  * @returns {boolean}
  */
-module.exports = (givenDate, now = new Date()) => {
+module.exports = (givenDate, now = new Date(), appealType = APPEAL_ID.HOUSEHOLDER) => {
```

Comment:
Should we be setting default values in the validation functions?

Reply:
Default value needs to be set as call further down the line done this way